modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicReaderTest.java

```diff
@@ -1241,7 +1241,6 @@ public void timeAdditionalDimRanges() throws Exception {
 
     /** Simple test method to test emptyMosaic creation support followed by harvesting. dataset */
     @Test
-    // @Ignore
     public void testEmpytMosaic() throws Exception {
```

Comment:
(not your fault, but while you're at it...)

```suggestion
    public void testEmptyMosaic() throws Exception {
```